boards/nrf52840-dongle/Makefile.dep

```diff
@@ -0,0 +1,25 @@
+ifneq (,$(filter saul_default,$(USEMODULE)))
+  USEMODULE += saul_gpio
+endif
+
+# It'd be nice to pull this in on any stdin dependency and not just shell, but
```

Comment:
Here and elsewhere: Do you mean `stdio`?

```suggestion
# It'd be nice to pull this in on any stdio dependency and not just shell, but
```

Reply:
Yes, fixup'ed.